### PR TITLE
insighttoolkit 4.7.1

### PR DESCRIPTION
**Pull request by iMichka**
Updated to ITK 4.7.1

I have great news. The Python wrapping works again (after a lot of work done fixing it during the last year). For the moment one can only build it using the HEAD version. The next stable version (ITK 4.8.0) will bring this feature back.

### CODE REVIEW

File: insighttoolkit.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 class Insighttoolkit < Formula
   homepage 'http://www.itk.org'
-  url 'https://downloads.sourceforge.net/project/itk/itk/4.6/InsightToolkit-4.6.1.tar.gz'
-  sha1 '95b879b17a8e2c4727b363037a3af4377095422d'
+  url 'https://downloads.sourceforge.net/project/itk/itk/4.7/InsightToolkit-4.7.1.tar.gz'
+  sha1 '70815c884b82899c51c7563296ba4b9ac0bf5e26'
   head 'git://itk.org/ITK.git'
   revision 1
```
Comment (iml) [inline insighttoolkit.rb]: Remove the revision when upgrading to new versions.

File: insighttoolkit.rb [hunk 2]
```diff
@@ -33,11 +33,10 @@ class Insighttoolkit < Formula
   option 'remove-legacy', 'Disable legacy APIs'
   option 'with-review', 'Enable modules under review'
 
-  if build.with? 'python'
+  if build.with? 'python' and !build.head?
```
Comment (iml) [inline insighttoolkit.rb]: `if build.with?("python") && build.stable?` See `brew audit --strict`

**Comment by iMichka** [comment]
Brew audit should be happier now :)

**Comment by iMichka** [comment]
The yosemite build passed, the two others were aborted (don't know why). Did you only trigger one build ?

Homebrew Science Pull Requests » mountain_lion completed with result ABORTED
Homebrew Science Pull Requests » yosemite completed with result SUCCESS
Homebrew Science Pull Requests » mavericks completed with result ABORTED

**Comment by iml** [comment]
The bot currently times out after an hour, so VTK couldn't be bottled and as this depends on VTK it also times out. Since Yosemite gets in just under the wire, it still might be possible to bottle things again with some tweaking. Otherwise this will need to be pulled sans bottle.

**Comment by iMichka** [comment]
Maybe we can get this in without bottles for the moment.
Does the time out include the build of all dependencies ? Maybe it should only take into account the build time of the updated package (though that may be difficult to implement). Or would it possible to "whitelist" ITK, as this is a package known to need some time to build. And those big packages are then also especially interesting to have as bottles to speed up installation :)

**Comment by iml** [comment]
The timeout is per-machine for each PR, including dependencies. I'm not sure we can make exceptions at this point but @mikemcquaid would be able to answer that. The bots have been pretty busy recently. I'll go ahead pull this with just the Yosemite bottle, though.

**Comment by mikemcquaid** [comment]
We can't really make exceptions for now because the bot machines are having issues. Once they get resolved the speed will increase.

**Pull request closed without merging**